### PR TITLE
docs(queries): update filter, pagination and sorting details for assorted queries

### PR DESCRIPTION
**Pull request by rahulkeerthi**
Changes:
- add search filter details to `get patients` query
- update details of pagination and sorting for `get patients`, `search activities`, `get pathways`

### CODE REVIEW

File: content/awell-orchestration/api-reference/queries/get-pathways.mdx [hunk 1]
```diff
@@ -86,7 +86,7 @@ When specifying multiple filters, the resulting criteria is the conjunction of a
 
 ### Pagination and sorting
 
-Pagination and sorting are highly recommended. Not using pagination means that the API will return all pathways at once which will cause performance issues.
+Pagination and sorting are required for this query. If no pagination values are supplied, the API will return only the first 10 care flows, or 10 care flows from any specified offset.
```
Comment (nckhell) [inline content/awell-orchestration/api-reference/queries/get-pathways.mdx:89]: [C] We need to make sure this is listed as a breaking change in the corresponding changelog. Can you create a placeholder for the corresponding release page in the changelog as well as part of this MR?

File: content/awell-orchestration/api-reference/queries/get-patients.mdx [hunk 2]
```diff
@@ -67,41 +69,43 @@ query GetPatients(
...
+      "sort_field": "last_name",
+      "sort_direction": "desc"
+    }
   `}
 </code>
 
 ### Pagination and sorting
 
-Pagination and sorting are highly recommended. Not using pagination means that the API will return all activities at once which will cause performance issues.
+Pagination and sorting are required for this query. If no pagination values are supplied, the API will return only the first 10 patients, or 10 patients from the specified offset.
```
Comment (nckhell) [inline content/awell-orchestration/api-reference/queries/get-patients.mdx:93]: Same comment: make sure it is listed as a breaking change on changelog page.

File: content/awell-orchestration/api-reference/queries/search-activities.mdx [hunk 3]
```diff
@@ -95,9 +95,9 @@ When specifying multiple filters, the resulting criteria is the conjuction of al
 
 ### Pagination and sorting
 
-Pagination and sorting are highly recommended. Not using pagination means that the API will return all activities at once which will cause performance issues.
+Pagination and sorting are required for this query. If no pagination values are supplied, the API will return only the first 10 activities, or 10 activities from the specified offset.
```
Comment (nckhell) [inline content/awell-orchestration/api-reference/queries/search-activities.mdx:98]: Same comment.

**Review by nckhell — CHANGES_REQUESTED** [review]

**Comment by nckhell** [comment]
@rahulkeerthi Do you make sure to update this MR and address the comment made here in Github and [here](https://gitlab.com/awellhealth/awell-next/-/merge_requests/2677/)?

**Comment by rahulkeerthi** [comment]
@nckhell changes made, i've set it as release v1.20.6 (which is Monday's release)

**Review by nckhell — APPROVED** [review]

**Comment by nckhell** [comment]
@rahulkeerthi can you merge this?

**Review by nckhell — APPROVED** [review]

**Pull request merged**